src/applications/static-pages/ezr-tera-alert/components/App/index.unit.spec.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import { App } from '.';
+
+describe('ezr submission options', () => {
```

Comment:
These tests do not reflect this app. Please update to reflect a meaningful test of the conditionals related to the content for this app.